test/parallel_api/ranges/range.zip/zip_view.ctad.compile.pass.cpp

```diff
@@ -0,0 +1,64 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+// template <class... Rs>
+// zip_view(Rs&&...) -> zip_view<views::all_t<Rs>...>;
+
+#include "support/test_config.h"
+#include "support/utils.h"
+
+#if _ENABLE_STD_RANGES_TESTING
+
+#include <cassert>
+#include <utility>
+
+#include <oneapi/dpl/ranges>
+#include <ranges>
+
+namespace dpl_ranges = oneapi::dpl::ranges;
+
+struct Container {
+  int* begin() const;
+  int* end() const;
+};
+
+struct View : std::ranges::view_base {
+  int* begin() const;
+  int* end() const;
+};
+
+#if __GNUC__ && _ONEDPL_GCC_VERSION >= 120100
```

Comment:
Why does it need this GCC version? Should we test it if it is not libstdc++?

Reply:
Basically, the same guard was in the original LLVM test. Probably, there was a bug for earlier GCC versions.
I just adapted macro name  - `_ONEDPL_GCC_VERSION`.

Reply:
There is no such macro in the original [range.zip/ctad.compile.pass.cpp](https://github.com/llvm/llvm-project/blob/main/libcxx/test/std/ranges/range.adaptors/range.zip/ctad.compile.pass.cpp).

Reply:
Let's correct the macro then

Reply:
> correct the macro

What do you mean?